README.md

```diff
@@ -50,3 +92,12 @@ Licensed under the GPLv2
     You should have received a copy of the GNU General Public License
     along with this program; if not, write to the Free Software
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+See the [COPYING][copying] file for the full license information.
+
+[ratbagd-readme]: https://github.com/libratbag/libratbag/blob/master/README.md#running-ratbagd-as-dbus-activated-systemd-service
+[meson]: http://mesonbuild.com/
+[ninja]: https://ninja-build.org/
+[kanban]: https://github.com/orgs/libratbag/projects/1
+[wiki-redesign]: https://github.com/libratbag/piper/wiki/Piper-Redesign
+[copying]: https://github.com/libratbag/piper/blob/master/COPYING
```

Comment:
Not sure if they'd be handled correctly, but I find the plain-text markdown easier to read this way. Will change it though